okta/request_executor.py

```diff
@@ -118,8 +118,8 @@ async def create_request(self, method: str, url: str, body: dict = None,
 
         # Build request
         # Get predetermined headers and build URL
-        headers.update(self._custom_headers)
-        headers.update(self._default_headers)
+        headers = {**self._custom_headers, **headers}
+        headers = {**self._default_headers, **headers}
```

Comment:
The **syntax destructures the dictionaries and prioritizes entries in the `headers` parameter set by the user